test/es6/rlexe.xml

```diff
@@ -689,6 +689,18 @@
   </test>
   <test>
     <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -args summary -endargs</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -nodeferparse -args summary -endargs</compile-flags>
```

Comment:
I added it there to specifically test cases where the body symbols are not forced into slots. First one anyway does add some defer parse scenario when ran in the suite.